core/app/models/spree/taxon.rb

```diff
@@ -79,12 +79,32 @@ def to_param
       permalink
     end
 
+    # @return [Array<Integer>] the ids of descendant taxons.
+    def descendant_ids
```

Comment:
I don't think this is really worth being a public method, its more likely anyone that wants this will not know it exists and call `descendants.pluck(:id)`